docs/Hosts/Spawn-Hosts.md

```diff
@@ -50,14 +50,34 @@ Clicking it will pre-populate the spawn host page with a request to spawn a host
 
 ![spawn_host_modal.png](../images/spawn_host_modal.png)
 
-Fetching artifacts can also be performed manually; see [fetch](../CLI#fetch) in the Evergreen command line tool documentation.
+The spawn host will not fetch the binaries and artifacts associated with the task on start-up because it lacks the necessary permissions. After you SSH into the host, you will need to run the following command to fetch them (which will prompt you to authenticate):
+
+```sh
+evergreen host fetch
```

Comment:
I think we should consider using something more specific like `evergreen host fetch-task-data` in case we want to have a more generic `evergreen host fetch` for some other feature.